app/assets/javascripts/student_profile_v2/student_profile_header.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+  window.shared || (window.shared = {});
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+  var Routes = window.shared.Routes;
+
+  var styles = {
+    titleContainer: {
+      fontSize: 16,
+      padding: 20
+    },
+    nameTitle: {
+      fontSize: 20,
+      fontWeight: 'bold'
+    },
+    titleItem: {
+      padding: 5
+    }
+  };
+
+  /*
+  This pure UI component renders top-line information like the student's name, school and
+  classroom.
+  */
+  var StudentProfileHeader = window.shared.StudentProfileHeader = React.createClass({
+    displayName: 'StudentProfileHeader',
+
+    propTypes: {
+      student: React.PropTypes.object.isRequired
+    },
+
+    render: function() {
+      var student =  this.props.student;
+      return dom.div({ style: styles.titleContainer },
```

Comment:
If you Google for React examples, you'll find JSX that looks like HTML with brackets.  This works the same way but is just using plain JavaScript syntax instead.